### PR TITLE
ci: Enable nightly build for Windows MSI and integrate atom action into 1 click automation

### PR DESCRIPTION
**Pull request by KevinLiAWS**
Enable nightly build for Windows MSI and integrate atom action into 1 click automation

*Description of changes:*
Added the scheduled time to Windows build and test action.

*Testing done:*
TODO

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/build-and-test-msi.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ on:
       ref_name:
         required: true
         type: string
+  schedule:
+    - cron: '0 9 * * *'
```
Comment (vsiravar) [inline .github/workflows/build-and-test-msi.yaml:12]: Maybe a comment about running this job at 9:00 UTC will help decoding the cron expression.
# builds windows installer and runs tests daily at 9:00 UTC.

File: .github/workflows/release-automation.yaml [hunk 2]
```diff
@@ -39,3 +39,20 @@ jobs:
...
+      ref_name: ${{ needs.get-latest-tag.outputs.tag }}
+  
+  upload-msi-to-release:
+      needs:
+      - get-latest-tag
+      - build-and-test-finch-msi
+    uses: ./.github/workflows/upload-msi-to-release.yaml
+    secrets: inherit
+    with:
+      ref_name: ${{ needs.get-latest-tag.outputs.tag }}
```
Comment (vsiravar) [inline .github/workflows/release-automation.yaml:58]: nit: deleted new line.

File: .github/workflows/release-automation.yaml [hunk 3]
```diff
@@ -39,3 +39,20 @@ jobs:
...
+  build-and-test-finch-msi:
+    build-and-test-finch-pkg:
+    needs: get-latest-tag
+    uses: ./.github/workflows/build-and-test-msi.yaml
+    secrets: inherit
+    with:
+      ref_name: ${{ needs.get-latest-tag.outputs.tag }}
+  
+  upload-msi-to-release:
+      needs:
```
Comment (KevinLiAWS) [inline .github/workflows/release-automation.yaml:52]: Dense for this line is wrong

File: .github/workflows/lint-pr-title.yaml [hunk 4]
```diff
@@ -14,7 +14,11 @@ jobs:
     name: conventional-commit
     runs-on: ubuntu-latest
     steps:
+<<<<<<< HEAD
```
Comment (KevinLiAWS) [inline .github/workflows/lint-pr-title.yaml:17]: Seems have unresolved conflict

File: .github/workflows/release-automation.yaml (hunk 3 above)
Comment (pendo324) [reply .github/workflows/release-automation.yaml:52]: Thanks, seems like I messed up some of the merge conflicts. Will push fixes after tests run.

File: .github/workflows/lint-pr-title.yaml (hunk 4 above)
Comment (pendo324) [reply .github/workflows/lint-pr-title.yaml:17]: Made the change locally but somehow didn't commit it before merging the merge commit :(

File: pkg/winutil/run_windows.go [hunk 5]
```diff
@@ -0,0 +1,210 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (vsiravar) [inline pkg/winutil/run_windows.go:1]: Do we still need this file?

Comment (pendo324) [reply pkg/winutil/run_windows.go:1]: No, I've got a change pending to remove it, just want to see how this windows e2e test run goes (in reality, its not testing anything new for this PR, I'm just curious).

**Comment by pendo324** [comment]
All comments should be resolved now @KevinLiAWS @vsiravar

**Review by pendo324 — APPROVED** [review]

**Comment by pendo324** [comment]
Failing tests are unrelated, overriding and merging.

**Pull request merged**